docker/builder/build-node.sh

```diff
@@ -11,20 +11,28 @@ echo "PROFILE: $PROFILE"
 echo "FEATURES: $FEATURES"
 echo "CARGO_TARGET_DIR: $CARGO_TARGET_DIR"
 
-# Build and overwrite the aptos-node binary with features if specified
-if [ -n "$FEATURES" ]; then
-    echo "Building aptos-node with features ${FEATURES}"
-    cargo build --profile=$PROFILE --features=$FEATURES -p aptos-node "$@"
-else 
-    # Build aptos-node separately
-    cargo build --locked --profile=$PROFILE \
-        -p aptos-node \
-        "$@"
-fi
+PACKAGES=(
+    aptos-node
+    aptos-forge-cli
+)
```

Comment:
I would just inline it and build both together, without the for loop.

Reply:
This is what I had originally done, however this results in the aptos-node panic on startup because when you invoke them together it unifies the target features which leads to an invalid set of features enabled for aptos-node!

The comment above the for loop describes this a bit